core/src/main/java/com/graphhopper/reader/dem/AbstractEdgeElevationInterpolator.java

```diff
@@ -0,0 +1,114 @@
+package com.graphhopper.reader.dem;
+
+import com.graphhopper.coll.GHBitSet;
+import com.graphhopper.coll.GHBitSetImpl;
+import com.graphhopper.routing.util.AllEdgesIterator;
+import com.graphhopper.routing.util.DataFlagEncoder;
+import com.graphhopper.storage.GraphHopperStorage;
+import com.graphhopper.util.BreadthFirstSearch;
+import com.graphhopper.util.EdgeExplorer;
+import com.graphhopper.util.EdgeIterator;
+import com.graphhopper.util.EdgeIteratorState;
+import com.graphhopper.util.PointList;
+
+import gnu.trove.set.TIntSet;
+import gnu.trove.set.hash.TIntHashSet;
+
+public abstract class AbstractEdgeElevationInterpolator {
```

Comment:
Would you mind to add one sentence description to every new class. I know the current state is not always this way but we should improve inline documentation a bit :)